stories/topics/proc-aws-marketplace.adoc

```diff
@@ -25,6 +25,8 @@ If you have worked with Red Hat Sales to enable a private offer of {AAPonAWS}, f
 .Procedure
 . Visit your Private Offer with the URL link provided by your Red Hat Sales representative.
 . Click btn:[Accept Terms] to subscribe to the AMI Private Offer named {PlatformNameShort} from {Market}.
+.. If you are accepting terms for the Ansible Automation Platform foundation offer, then you will want to accept the discount rate for three virtual machines.  All Ansible Automation Platform foundation deployments use three VMs.  *If you accept fewer than three VMs, then you will not receive the proper discount rate for your deployment.*
```

Comment:
{PlatformNameShort} instead of Ansible Automation Platform.

Also remove the * from the last sentence, we don't emphasis like that.

If it's important enough, use
[NOTE]
====
If you accept fewer than three VMs,  you will not receive the proper discount rate for your deployment.
====

Reply:
These two bullets are actually critical for the sales process.  It became awkward with the sub bullets and a separate node, so I moved the content to the notes section instead.